site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -269,7 +269,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.readiness.initialDelaySeconds`      | Initial delay before performing the first probe (in seconds)                                                                                                                                                            | `3`                                |
 | `agones.allocator.readiness.periodSeconds`            | Seconds between every liveness probe (in seconds)                                                                                                                                                                       | `3`                                |
 | `agones.allocator.readiness.failureThreshold`         | Number of times before giving up (in seconds)                                                                                                                                                                           | `3`                                |
-| `agones.system.requireDedicatedNode`                  | Forces Agones system components to be scheduled on dedicated nodes                                                                                                                                                                           |  `false`                           |
+| `agones.system.requireDedicatedNode`                  | Forces Agones system components to be scheduled on dedicated nodes, only applies to the GKE Standard mode cluster                                                                                                                                                                          |  `false`                           |
```

Comment:
Why is that true? It should be doable: https://cloud.google.com/kubernetes-engine/docs/how-to/workload-separation

Reply:
On that page, it's not clear to me whether we can use same mechanism as today (node affinity) on AP. The error I got when apply the install.yaml is 
~~~
node affinity is not allowed on labels with keys: 'agones.dev/agones-system'; Autopilot only allows labels with keys: 'cloud.google.com/compute-class,cloud.google.com/gke-spot,topology.kubernetes.io/region,topology.kubernetes.io/zone,failure-domain.beta.kubernetes.io/region,failure-domain.beta.kubernetes.io/zone,cloud.google.com/gke-os-distribution,kubernetes.io/os,kubernetes.io/arch,cloud.google.com/private-node,sandbox.gke.io/runtime,cloud.google.com/gke-accelerator,cloud.google.com/gke-accelerator-count,iam.gke.io/gke-metadata-server-enabled'."
~~~

So my guess is, on AP, we can only use `spec.nodeSelector`.

Reply:
We may need to add a statement somewhere to call out using `agones.*.nodeSelector` for separating agones system on AP.

Reply:
That's unfortunate. What we have right now is the recommended way to do it for GKE clusters w/o node autoprovisining: https://cloud.google.com/kubernetes-engine/docs/how-to/isolate-workloads-dedicated-nodes

It looks like on autopilot you use the node selector _instead_ of affinity. And it seems like even with a node selector, if we specify affinity the spec will be rejected. That's concerning, because with this change as-is, we now always have affinity specified (before you could override it to turn it off).

Reply:
Should we back off on this change? I feel like we could rework this into something that would work for both Autopilot and Standard.

Reply:
Agreed. I'm looking into nodeSelector

Reply:
There is no common way to do workload isolation on Autopilot and Standard without auto-provisioning. I'm going to merge this for now. The instructions on isolate workload on Autopilot should be in separate PR.

Reply:
I think the documentation change is incorrect then? It still says it only applies to Standard.

Reply:
I changed it to "...only applies to the GKE Standard without node auto-provisioning" in last revision.